time.cabal

```diff
@@ -151,3 +150,66 @@ test-suite tests
         Test.AddDays
         Test.AddDaysRef
         Test.TestUtil
+        
+benchmark bench
+    hs-source-dirs: lib, benchmark
```

Comment:
Since time is a bundled library, i think what we really want to do here is remove `time` from build depends, then this benchmark will not interference with system `time`. But if there's simpler way to do it then i'll make a change.